pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -43,7 +41,7 @@ func NewPmemDeviceManagerLVM() (PmemDeviceManager, error) {
 
 	ctx, err := ndctl.NewContext()
 	if err != nil {
-		return nil, fmt.Errorf("Failed to initialize pmem context: %s", err.Error())
+		return nil, fmt.Errorf("%w occurred while initialize pmem context: %s", ErrUnknown, err.Error())
```

Comment:
All of the new `fmt.Errorf` should follow the Go idiom of `<description>: <inner error>`. We don't need full sentences either.

For example, in this case `"pmem context: %w: %v", ErrUnknown, err` is better.

Note also the use of `%v` and `err` instead of `%s` and `err.Error()`: it's the idiomatic approach and shorter.

I'm not going to mark further occurrences of this. Please check the diff and update.

Reply:
Done. Rephrased all error messages as suggested.